pkg/operator/starter.go

```diff
@@ -0,0 +1,107 @@
+package operator
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/openshift/aws-efs-csi-driver-operator/assets"
+	"github.com/openshift/library-go/pkg/operator/loglevel"
+	"github.com/openshift/library-go/pkg/operator/managementstatecontroller"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	opv1 "github.com/openshift/api/operator/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	configinformers "github.com/openshift/client-go/config/informers/externalversions"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/controller/manager"
+	"github.com/openshift/library-go/pkg/operator/csi/csidrivernodeservicecontroller"
+	goc "github.com/openshift/library-go/pkg/operator/genericoperatorclient"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	// Operand and operator run in the same namespace
+	operatorName = "aws-efs-csi-driver-operator"
+	instanceName = "efs.csi.aws.com"
```

Comment:
We could get rid of this const by the one from `opv1` package.

Reply:
Fixed